pom.xml

```diff
@@ -56,13 +56,13 @@
             </dependency>
             <dependency>
                 <groupId>edu.ucar</groupId>
-                <artifactId>netcdf</artifactId>
-                <version>4.3.22</version>
+                <artifactId>netcdf4</artifactId>
+                <version>4.6.6</version>
             </dependency>
             <dependency>
                 <groupId>edu.ucar</groupId>
                 <artifactId>opendap</artifactId>
-                <version>4.3.22</version>
+                <version>4.6.6</version>
```

Comment:
Bear in mind that the servers are stuck on the 4.1.1 system library. Assuming these are just the Java bindings versions, do they still work as expected on our old libnetcdf6 version?